src/Features/CSharp/Portable/ImplementInterface/AbstractChangeImplementationCodeRefactoringProvider.cs

```diff
@@ -71,8 +71,11 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
             // (like "GetEnumerator"), instead of the auto-generated names that the compiler makes
             // like: "System.IEnumerable.GetEnumerator"
             directlyImplementedMembers.AddRange(member, member.ExplicitOrImplicitInterfaceImplementations());
+            var implementations = member.ExplicitOrImplicitInterfaceImplementations();
+            if (implementations.IsEmpty)
+                return;
```

Comment:
```suggestion
            var implementations = member.ExplicitOrImplicitInterfaceImplementations();
            if (implementations.IsEmpty)
                return;

            directlyImplementedMembers.AddRange(member, implementations);
```

Reply:
nit: declaration of 'directlyImplementedMembers' could be moved after the return too